velox/exec/HashTable.h

```diff
@@ -425,6 +425,10 @@ class HashTable : public BaseHashTable {
   /// purpose.
   void checkConsistency() const;
 
+  void testingSetHashMode(HashMode mode, int32_t numNew) {
```

Comment:
Why exactly we need a method that just calls another method?
Because that another method is private?

Reply:
Yes, it is not supposed to be used in production and the hash mode is decided by the data distribution. So I just set it to hash mode to ease testing here.

Reply:
@xiaoxmeng 
We should name such methods more explicitly.
For instance:

`testingOnly_SetHashMode()`

Reply:
Yeah, I just keep the test method naming to be consistent with the others in Velox code base. testingOnly_SetHashMode is better or TEST_testMethodName as Google's coding style suggests.